kubectl-pg/cmd/util.go

```diff
@@ -0,0 +1,61 @@
+package cmd
+
+import (
+	"flag"
+	"fmt"
+	restclient "k8s.io/client-go/rest"
+	"k8s.io/client-go/tools/clientcmd"
+	"k8s.io/client-go/util/homedir"
+	"log"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"strings"
+)
+
+func getConfig() *restclient.Config {
+	var kubeconfig *string
+	var config *restclient.Config
+	envKube := os.Getenv("KUBECONFIG")
+	if  envKube != ""{
+		kubeconfig = &envKube
+	} else {
+		if home := homedir.HomeDir(); home != "" {
+			kubeconfig = flag.String("kubeconfig", filepath.Join(home, ".kube", "config"), "(optional) absolute path to the kubeconfig file")
+		} else {
+			kubeconfig = flag.String("kubeconfig", "", "absolute path to the kubeconfig file")
+		}
+	}
+		flag.Parse()
+		var err error
+		config, err = clientcmd.BuildConfigFromFlags("", *kubeconfig)
+		if err != nil {
+			log.Fatal(err)
+		}
+		return config
+}
+
+func getCurrentNamespace() string {
+	namespace, err := exec.Command("kubectl", "config", "view", "--minify", "--output", "jsonpath={..namespace}").CombinedOutput()
+	if err != nil {
+		log.Fatal(err)
+	}
+	currentNamespace := string(namespace)
+	if currentNamespace == "" {
+		currentNamespace = "default"
+	}
+	return currentNamespace
+}
+
+func confirmAction(clusterName string, namespace string) {
+	for {
+		confirmClusterDetails := ""
+		_, _ = fmt.Scan(&confirmClusterDetails)
```

Comment:
please do not ignore errors with `_`,  it is generally a bad practice. Inform the user something went wrong instead.

Reply:
added an error check here